src/main/resources/users.properties

```diff
@@ -0,0 +1,5 @@
+# List of users
+users.list=idsUser
```

Comment:
Please add here some names, Bob, Alice, so we show that there can be multiple users

Reply:
It is in detail explained how to add a new user in README.md, but I'll add it here too.